### PR TITLE
feat(automations): add RLS_YEAR release-year condition

### PR DESCRIPTION
**Pull request by nitrobass24**
## Summary

Adds a new automation condition field, **`RLS_YEAR`** ("Year (RLS)"), that filters torrents by the release year parsed from the torrent name. The value comes from the existing `rls` release-name parser (`pkg/releases`) — the same parser already powering the sibling `RLS_*` fields (source, resolution, codec, group, …) — so there's no new parsing, no external lookups, and no schema changes.

## Behavior

- **Numeric, filter-only.** Full numeric operator set (`=`, `!=`, `>`, `>=`, `<`, `<=`, `between`). Intentionally **not** available as a sort/score field.
- **Unknown year = no match.** When no 4-digit year can be parsed (year `0`), the condition never matches — for *any* operator, including `is not`. Mirrors the existing `compareAgeIfSet` guard for unset timestamps and avoids foot-guns (e.g. `RLS_YEAR < 2000 → delete` won't sweep up yearless releases).
- **Best for movies / dated releases.** Most TV episodes (e.g. `S14E05`) carry no year and won't match; only releases with an explicit 4-digit year (movies, dated pilots like `S01E01.1999`, or episodes tagged `(2012)`) get a usable value.
- **Save-time validation.** Years outside `1900..currentYear+1` are rejected (the parser does no range sanity check), including `between` bounds.

## Implementation

- **Backend:** new `FieldRlsYear` field, `torrentRlsYear` extractor, `compareRlsYearIfSet` evaluator guard, `validateRlsYearConditions` range validation.
- **Frontend:** new field in the query-builder constants/type union and the "Release" group; numeric input + operators come for free from the `integer` field type.
- **i18n:** labels for `en` / `de` / `fr` / `zh-CN`.
- **Docs:** new row in the automations condition-field reference.

No DB migration (conditions are stored as JSON) and no OpenAPI change (conditions are inline JSON, no field enum) required.

## UI

Adds a **"Year (RLS)"** option under the **Release** group in the condition field picker. No other visual changes.

## Testing

- [x] `go test -race -count=1 ./internal/services/automations/... ./internal/api/handlers/...` — incl. new `TestEvaluateCondition_RlsYear` (9 cases) and `TestValidateRlsYearConditions` (13 cases)
- [x] `pnpm check:i18n` — 0 errors (all 4 locales in parity)
- [x] `make precommit` — 0 errors
- [x] `make build`
- [x] Verified parsing on sample names (incl. `Law and Order ... S14E05 ... 264-playWEB` → year `0`/no-match) via the `rls` parser
- [x] Live test in a running instance (recommended before merge)

## Summary by CodeRabbit

* **New Features**
  * Added **Release Year (RLS_YEAR)** as a selectable automation condition with equals, greater-than/less-than, and inclusive between support.
* **Bug Fixes**
  * Improved validation for malformed/out-of-range year values and invalid `between` bounds (including fractional years), returning clear HTTP 400 errors.
  * Yearless releases never match year comparisons (including `!=`), and numeric condition values now ignore surrounding whitespace.
* **Documentation**
  * Updated Automations Query Builder docs describing year parsing, match behavior, and how NOT affects the condition.
* **Localization**
  * Added **RLS_YEAR** translations in multiple languages (EN/DE/FR/ZH-CN/IT/KO).
* **Tests**
  * Added unit tests for RLS year parsing, evaluation, and validation edge cases.

### CODE REVIEW

File: documentation/docs/features/automations.md [hunk 1]
```diff
@@ -158,6 +158,7 @@ Note: if you have **Settings → Tracker Customizations** configured, the **Trac
 | Release Audio      | Parsed release specifier (e.g. `TrueHD`; may be empty)                                     |
 | Release Channels   | Parsed release specifier (e.g. `5.1`; may be empty)                                        |
 | Release Group      | Parsed release specifier (e.g. `NTb`; may be empty)                                        |
+| Release Year       | Year parsed from the torrent name (e.g. `2021`). Numeric; best for movies/dated releases. Releases with no detectable year — including most TV episodes (`S14E05`) — never match, for any operator (including `is not`) |
```
[the remark this answers is not in the document]
Comment (nitrobass24) [reply documentation/docs/features/automations.md]: 5fad7d2a

File: internal/api/handlers/automations.go [hunk 2]
```diff
@@ -686,6 +691,69 @@ func validateConditionTreeGroupIDs(cond *models.RuleCondition, knownGroupIDs map
...
+	if cond.Operator == models.OperatorBetween {
+		if cond.MinValue == nil || cond.MaxValue == nil {
+			return "Release Year range requires both a minimum and maximum year", errors.New("release year range missing bound")
+		}
+		if *cond.MinValue > *cond.MaxValue {
+			return "Release Year minimum cannot be greater than maximum", errors.New("release year min greater than max")
+		}
+		if !inRange(*cond.MinValue) || !inRange(*cond.MaxValue) {
+			return rangeMsg, errors.New("release year out of range")
+		}
```
[the remark this answers is not in the document]
Comment (nitrobass24) [reply internal/api/handlers/automations.go:747]: 5fad7d2a

**Comment by nitrobass24** [comment]
@coderabbit review
